integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/projection/FieldSearchProjectionIT.java

```diff
@@ -288,6 +288,44 @@ public void withProjectionConverters_rawValues() {
 		}
 	}
 
+	@Test
+	public void withDslConverters_rawValues() {
```

Comment:
The tests shouldn't be about DSL converters. They should be about projecting with `field()` when no projection converter is defined (this should fail) or projecting with `rawField()` when no projection converter is defined (this should work).